src/core/request.js

```diff
@@ -66,8 +66,9 @@ var createError = function createError(response, fallbackMessage, httpCode, http
     }
     err.response = response;
     err.sent = httpSent;
+    err.source = response.source || 'request';
 
-    if (httpCode) {
+    if (_.isNumber(httpCode)) {
```

Comment:
Is it necessary to use lodash here? maybe a simple !isNaN could be enough.

Reply:
Both are used extensively in TAO; in this file for consistency I prefer to stick with the `_.isNumber()`.

Reply:
Ok, indeed it would make sense to remove if this would be the only usage of lodash.